### DISCUSSION TITLE
Front matter summary vs <!--more--> tag - behavior differs

### DISCUSSION THREAD

**Discussion by TymekDev** [adityatelange/hugo-PaperMod#312] [Q&A]
I am having trouble using the manual summary splitting tag - `<!--more-->`. Find the two examples below.

**Version A - post**
```
---
title: "Some title"
date: 2021-03-20
author: John Doe
summary: Some summary
---
Post content
```

**Version A - result**
![image](https://user-images.githubusercontent.com/38053499/111885807-65d83780-89ca-11eb-97ba-260d712c3f43.png)

**Version B - post**
```
---
title: "Some title"
date: 2021-03-20
author: John Doe
---
Some summary<!--more-->
Post content
```

**Version B - result**
![image](https://user-images.githubusercontent.com/38053499/111885857-94561280-89ca-11eb-9539-58b4711d45b7.png)

I have read content [summaries documentation](https://gohugo.io/content-management/summaries/) as well as [`.Truncated` page variable's description](https://gohugo.io/variables/page/#page-variables). I do not understand why for Version B `.Truncated` is true and not the same as in Version A.

I would love to have summary as part of the post and not in the front matter while not having the dots displayed. Any help is appreciated!

**Comment by adityatelange** [comment]
I think you are correct, but I didn't actually get why it does that ..

With the following content and nothing after `<!--more-->` `.Truncated` is false, but not with content after `<!--more-->`
```md
Some summary
<!--more-->

```
